colossalai/tensor/colo_parameter.py

```diff
@@ -7,11 +7,12 @@
 
 from .colo_tensor import _convert_output
 
-WHITE_LIST_FUNCS = {torch.Tensor.__getitem__, torch.Tensor.is_floating_point}
+WHITE_LIST_FUNCS = {torch.Tensor.__getitem__}
+IGNORED_FUNCS = {torch.Tensor.is_floating_point, torch.nn.functional.embedding, torch.nn.functional.layer_norm, torch.addmm, torch.nn.functional.linear}
```

Comment:
` torch.nn.functional.embedding, torch.nn.functional.layer_norm, torch.addmm, torch.nn.functional.linear` should not be ignored